src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/scripted_fields_table/components/call_outs/call_outs.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import {
+  EuiCallOut,
+  EuiLink,
+  EuiSpacer,
+} from '@elastic/eui';
+
+export const CallOuts = ({
+  deprecatedLangsInUse,
+  painlessDocLink,
+}) => {
+  if (!deprecatedLangsInUse.length) {
+    return null;
+  }
+
+  return (
+    <div>
+      <EuiCallOut
+        title="Deprecation Warning"
+        color="danger"
+        iconType="cross"
+      >
+        <p>
+          We&apos;ve detected that the following deprecated languages are in use: {deprecatedLangsInUse.join(', ')}.
+          Support for these languages will be removed in the next major version of Kibana and Elasticsearch.
+          We recommend converting your scripted fields to <EuiLink href={painlessDocLink}>Painless</EuiLink>.
+        </p>
```

Comment:
Here's my suggested rewrite.  I'd prefer to see a more descriptive title than Deprecation Warning

Deprecated languages in use 

The following deprecated languages are in use:  {deprecatedLangsInUse.join(', ')}. Support for these languages will be removed in the next major version of Kibana and Elasticsearch.  Convert you scripted fields to  <EuiLink href={painlessDocLink}>Painless</EuiLink> to avoid any problems.